src/main/java/com/example/carservice/service/CarService.java

```diff
@@ -7,5 +7,5 @@
 public interface CarService {
 
     List<Car> getCars();
```

Comment:
Don't use in method naming already understaning things like Cars. You are already in CarService. Example: getAll or findAll